opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/ConfigurationHelp.java

```diff
@@ -174,6 +173,8 @@ private static Object getSampleValue(Method setter, Object defaultValue) {
             inm.add("f:user-specified-value");
             inm.add("d:user-specified-value");
             return inm;
+        } else if (paramType == Configuration.MeterRegistryType.class) {
```

Comment:
Why doesn't the `paramType.isEnum()` handling of line 178 work for this?

Reply:
It chooses the value of `PROMETHEUS` which does not work with indexer, making tests fail:
```
java.lang.RuntimeException: error setting sample value PROMETHEUS for public void org.opengrok.indexer.configuration.Configuration.setIndexerMeterRegistryType(org.opengrok.indexer.configuration.Configuration$MeterRegistryType)

	at org.opengrok.indexer.configuration.ConfigurationHelp.getSamples(ConfigurationHelp.java:91)
	at org.opengrok.indexer.configuration.ConfigurationHelpTest.shouldCreateReadableUsage(ConfigurationHelpTest.java:36)
...
```